README.md

```diff
@@ -1,3 +1,19 @@
-## Organizing a Collaborator Summit
+# Node.js Collaborator Summit
+
+The Node.js Foundation hosts regular summits for active contributors to the node.js project, this repository is being used to organize them.
+
+## Cost
+Free!
```

Comment:
Seems like a reasonable suggestion. Hopefully, he can throw a sentence in here for me to use. (if not I'm likely to just merge so we get SOMETHING in here for now)